toolkit/scripts/precache.mk

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Contains:
+#	- Tools to fully hydrate the rpm cache from upstream repos with the any versions that match local .spec files
+
+.PHONY: pre-cache clean-precache
+
+precache_state_dir = $(CACHED_RPMS_DIR)/precache
+precache_downloaded_files = $(precache_state_dir)/downloaded_files.txt
+precache_chroot_dir = $(precache_state_dir)/chroot
+
+$(call create_folder,$(precache_state_dir))
```

Comment:
Don't we need to create `precache_chroot_dir` as well?

Reply:
The tools actually complain if it already exists, the chroot library will create it (unless its a re-used chroot for docker environments etc.).